config/link-check/excluded-links.yml

```diff
@@ -0,0 +1,17 @@
+- 'http://localhost:8000**'
+- 'http://millionsongdataset.com/pages/getting-dataset/#subset'
+- 'https://marketplace.visualstudio.com/items?itemName=stkb.rewrap'
+- 'https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml'
+- 'https://remote.dvc.org/get-started'
+- 'https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&amp;id=24c0ecc49a'
+- 'https://www.meetup.com/San-Francisco-Machine-Learning-Meetup/events/264846847/'
+- 'https://www.reddit.com/r/MachineLearning/comments/bx0apm/d_how_do_you_manage_your_machine_learning/'
+- 'https://www.amazon.com/DevOps-Handbook-World-Class-Reliability-Organizations-ebook/dp/B01M9ASFQ3'
+- '**linkedin.com/in/**'
```

Comment:
we were testing them before? (had a custom handler as far as I remember). I would keep it if it was able detecting actual 404 from some "I'm busy" error code LI sends back

Reply:
The reason I ignored them is that LinkedIn always returns status code 999 specifically for profiles specifically, both with GET and HEAD and whether the profile exists or not.

Reply:
got it! that's unfortunate